SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -131,11 +147,19 @@ mount_overlayfs() {
                 mount_volatile_persistent_volume $volume $overlay_mount_with_cnt
                 volume_mount_map[$volume]=$overlay_mount_with_cnt
             fi
+
+           # If 'ro' is in the mount options, use default values for upper and
+           # work directories.
+            if [[ "$overlayfs_mnt_option" == *"ro"* ]]; then
```

Comment:
is it possible to have another option with the two letter ro match while it's not ro?
I am wondering since there's no concept of tokens here... Maybe split by space and then look through each token...